### PR TITLE
Fix Keras API gradient clipping

### PR DESCRIPTION
**Pull request by hkvision**
Fix https://github.com/intel-analytics/zoo/issues/214

@dding3 I suppose this time what I do should be equivalent to what you do in your PR. Please have a review. :)

### CODE REVIEW

**Comment by dding3** [comment]
LGTM

**Comment by hkvision** [comment]
@dding3 Updated. Have another review to see whether the logic is the same as yours?

**Comment by hkvision** [comment]
jenkins passed http://172.168.2.101:8080/job/ZOO-PR-Validation/364/

**Comment by hkvision** [comment]
jenkins passed http://172.168.2.101:8080/job/ZOO-PR-Validation/372/

**Comment by hkvision** [comment]
@jason-dai commented in your PR @dding3 `Change to clearGradientClipping`
I can help you do the change in this PR if you wish.

**Comment by dding3** [comment]
Sure. Feel free to go ahead.

**Comment by hkvision** [comment]
jenkins passed http://172.168.2.101:8080/view/ZOO-PR/job/ZOO-PR-Validation/384/

**Comment by hkvision** [comment]
synced with Ding, merged.
